tests/utils.py

```diff
@@ -256,7 +256,7 @@ async def compile_done():
         code = compiling.code
         return code == 204
 
-    await retry_limited(compile_done, 10)
+    await retry_limited(compile_done, 15)
```

Comment:
Without this `tests/server/test_compilerservice.py::test_server_recompile` fails.
I added 2 flamegraphs of the execution of this test to the commit but I don't find what could be taking so much longer.